src/App.js

```diff
@@ -9,8 +9,22 @@ export class App extends Component {
     movies: moviesFromServer,
   };
 
-  addMovie = (movie) => {
-    // put your code here
+  addMovie = ({ ...movieProperties }) => {
+    const { title, description, imgUrl, imdbUrl, imdbId } = movieProperties;
+
+    this.setState((prevState) => {
+      const newMovie = {
+        title,
+        description,
+        imgUrl,
+        imdbUrl,
+        imdbId,
+      };
+
+      return {
+        movies: [...prevState.movies, newMovie],
+      };
+    });
   };
```

Comment:
много лишних действий 
```suggestion
  addMovie = (movie) => this.setState((prevState) => ({
      movies: [...prevState.movies, movie],
    });
  };
```